src/core/utils/misc.cpp

```diff
@@ -260,7 +271,7 @@ int Utils::Misc::pythonVersion()
         if (output.isEmpty())
             output = python_proc.readAllStandardError();
         const QByteArray version_str = output.split(' ').last();
```

Comment:
Look at the previous comment, you can use the same code in the loop to test in Windows.
```
#if defined(Q_OS_UNIX)
versions = [python3, python2, python]
#else
versions = [python]
#endif
for ( version in versions ) {}
->check if the executable exists
->extract the version with pythonVersionComplete()
```